### PR TITLE
HOTT-1333: Tidy up tariff update model

### PR DESCRIPTION
**Pull request by willfish**
### Jira link

https://transformuk.atlassian.net/browse/HOTT-1333

### What?

I have added/removed/altered:

- [x] Removed unused crud from the tariff update model, adds missing specs, removes unused specs and generally tidy up tariff update behaviour

### CODE REVIEW

File: app/models/tariff_update.rb [hunk 1]
```diff
@@ -5,52 +5,13 @@ class TariffUpdate
...
-    end
-  end
-
-  def updated_at
-    date_attribute('updated_at')
-  end
-
   def applied_at
-    date_attribute('applied_at')
+    Date.parse(attributes['applied_at'].to_s)
```
Comment (alexdesi) [inline app/models/tariff_update.rb]: maybe  `to_s` is redundant.

File: app/models/tariff_update.rb [hunk 2]
```diff
@@ -5,52 +5,13 @@ class TariffUpdate
...
-        # Cache for 1 hour
-        TradeTariffFrontend::ServiceChooser.cache_with_service_choice(
-          'tariff_last_updated', expires_in: 1.hour
-        ) do
-          func.call
-        end
-      end
+  def self.latest_applied_import_date
+    TradeTariffFrontend::ServiceChooser.cache_with_service_choice('tariff_last_updated', expires_in: 23.hours) do
+      all.first.try(:applied_at) || Time.zone.today
```
Comment (alexdesi) [inline app/models/tariff_update.rb:12]: `first` should work as `all.first`, and it should be more efficient

**Review by alexdesi — APPROVED** [review]
👍 + two minor comments

File: app/models/tariff_update.rb (hunk 2 above)
Comment (willfish) [reply app/models/tariff_update.rb:12]: Not activerecord, sadly

**Pull request merged**